runtime/near-vm-runner/src/prepare/instrument_v3.rs

```diff
@@ -736,17 +736,16 @@ fn call_unstack_instrumentation(
     globals: u32,
 ) {
     if max_operand_stack_size != 0 || function_frame_size != 0 {
+        let mut func = func.instructions();
+        let Some(stack_charge) = max_operand_stack_size.checked_add(function_frame_size) else {
+            func.call(STACK_EXHAUSTED_FN).unreachable();
+            return;
+        };
```

Comment:
Shouldn't this be able pass in the fee computed for function prologue in the first place? They ought to be the same, right?

Reply:
Did some refactoring to reuse the computed charge and return early on overflow